packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -513,8 +508,8 @@ export class AccountResolver {
       input.email,
       input.newPassword,
       input.accountResetToken,
-      input.options,
-      headers
+      input.options
+      // headers, TODO: Not sure why this request fails when forwarding headers
```

Comment:
What does it fail with / is this related to any `headers` removal comments in #14867 (I didn't look too closely)? If it's not breaking anything I suppose we can just file a follow up.

Reply:
This fails to get the correct `Bearer` token. The result is that the request fails with no authentication token found. I'll file a follow up because this will cause our emails to not have the correct location (would show load balancer ip/location).